index.html

```diff
@@ -2,26 +2,25 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>FOSSASIA 2017 Singapore | Asia's Open Technology Event, Open Hardware, Free Software, Open Design, Open Data</title>
+        <title>FOSSASIA 2016 Singapore | Asia's Open Technology Event, Open Hardware, Free Software, Open Design, Open Data</title>
```

Comment:
@singhjp006 The website is for the 2017 conference, why have you changed the year ?